app.js

```diff
@@ -1,80 +1,88 @@
-// JS for booklist
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
 
-// Functions for local storage
-function getBooks() {
-  let books;
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
   }
 
-  return books;
-}
+  static setBooks(books) {
+    if (books.length > 0) {
+      localStorage.setItem('books', JSON.stringify(books));
+    }
+  }
+
+  static removeBook(index) {
+    const books = this.getBooks();
+
+    books.splice(index, 1);
```

Comment:
- [ ] [Optional] I would advise you to use slice instead of splice, this is because slice mutates the array whereas the splice method doesn't, changing array itself is a best practice instead of mutating it, because it will help you to not waste your resources